blog/_posts/2016-11-16-team-kindr3d-beyond-tellerrand-16.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Team kindr3d @ Beyond Tellerrand 16
+layout: post
+created_at: Tue Nov 16 2016
+permalink: blog/2016-11-16-team-kindr3d-beyond-tellerland-16
+current: blog
+author: kindr3d
+twitter: TKindr3d
+categories:
+- student-posts
+---
+
+### Team kindr3d @ Beyond Tellerrand 16
+
+
+Team kindr3d were lucky enough to get free tickets for this year [Berlin BTconf](https://beyondtellerrand.com/events/berlin-2016/speakers). We want to say thanks to every member of RGSoC team who fought for us and got us the golden tickets to this unique event which gave us a lot to think about.
+We also met [Nynne](https://twitter.com/nynnest), last year’s alumni and also our supervisor for a week this summer. And we had great conversations and exchanged our experience about our own RGSoC and being a female junior developer.
+
+
+![Don't Panic](https://saintsebastian.github.io/content/images/2016/11/FullSizeRender.jpg)
+*helpful advice*
+
+
+### What is it?
+
+
+It is a general front-end/web/design conference. This particular instance had an absolute minimum of slides with code. Do with that information what you will. Some may not be happy about overall *softness* of talk and topics, but there are definitely ideas worth sharing and pondering on. Ever since [ViewSource 2016](https://viewsourceconf.org/berlin-2016/), where people were friendly, talks were perspective changing, food and drinks were plentiful and delicious, and last week of Berlin summer warmed the hearts and MacBooks of attendees (in coalition with free bar of course), it was clear that what makes conference good is something elusive: atmosphere, vide, whatever you want to call it. These events do not exist in the vacuum, they are perceived in their immediate historical context with sprinkles of one's personal life events. And BTconf this year happened to be a very special snapshot of the very particular way of life.
+
+
+### What was happening?
+
+
+The moment [Sacha Judd](http://www.sachajudd.com) made a hall full of tech folk contemplate on [One Direction fandom](https://gimletmedia.com/episode/6-one-direction-larry-shippers/) it was on like donkey kong. It is rather unfair to identify this talk as the best, since there were many great ones, but this one hit us and our partner in crime Nynne directly in the feels. For most of the people on this room being a shipper, writing a fanfiction or cheesy fansite were not the direct life experiences, though they have been for is. But everyone realte to the small tragedy of people doing things well and being excited about doing them while not appreciating their own skill, effort and enthusiasm. Somehow this feels a lot like being a junior. [This talk is wonderful](https://vimeo.com/channels/beyondtellerrand/190832862) and you should watch it. We will always remember not to be ashamed of my passion and things we've made with love.
+
+
+![team kindr3d gif](https://files.slack.com/files-tmb/T0PT2TT6W-F317H07CY-f148695305/output_2_360.gif)
```

Comment:
this and the following image are both broken for me, and the link redirects me to a login screen for slack — it might mean that the images can only be accessed by someone who has access to the channel they were posted in.